src/style.css

```diff
@@ -1,3 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.box_empty {
+  display: flex;
+  height: 60px;
+  flex-grow: 1;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  justify-content: flex-end;
+  margin-left: 0;
+  margin-right: 50px;
+}
+
+.titles {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  list-style: none;
+  align-items: center;
+  display: flex;
+}
+
+.menu {
+  color: black;
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+}
+
+.menu:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 3px;
+  text-underline-offset: 0;
+  text-decoration-color: #00acdc;
+}
+
+.logo {
+  position: absolute;
+  z-index: 1;
+  top: 10px;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.menu-4:hover {
+  color: #00acdc;
+  text-decoration: none;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  display: inline-block;
+  margin-left: 0;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  transition: background-color 0.3s;
+  border-radius: 10px;
+  border-style: 10px solid;
```

Comment:
The 'border-style' property should not have a value like '10px solid'. It should only specify the style of the border, such as 'solid', 'dashed', etc. If you intended to set a border width, use 'border-width' instead.